eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/data/tagwrite/BufferedTagWriteManager.java

```diff
@@ -0,0 +1,432 @@
+package com.hms_networks.americas.sc.ignition.data.tagwrite;
+
+import com.hms_networks.americas.sc.ignition.comm.M2WebCommunicationManager;
```

Comment:
This cannot be resolved. Perhaps in a future commit?

Reply:
Correct, that will be in the following PR.